tests/api/Jalasoft.TeamUp.Resumes.API.Tests/GetResumeByIdTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Tests
+{
+    using System;
+    using System.Collections.Generic;
+    using Jalasoft.TeamUp.Resumes.API.Controllers;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Http.Internal;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Primitives;
+    using Moq;
+    using Xunit;
+
+    public class GetResumeByIdTests
+    {
+        private readonly Mock<IResumesService> mockService;
+        private readonly DefaultHttpContext mockHttpContext;
+        private readonly GetResumeById getResume;
+
+        public GetResumeByIdTests()
+        {
+            this.mockService = new Mock<IResumesService>();
+            this.mockHttpContext = new DefaultHttpContext();
+            this.getResume = new GetResumeById(this.mockService.Object);
+        }
+
+        [Fact]
+        public void GetResume_Returns_OkObjectResult()
+        {
+            // Arrange
+            var request = this.mockHttpContext.Request;
+            this.mockService.Setup(service => service.GetResume(Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"))).Returns(new Resume());
+
+            // Act
+            var response = this.getResume.Run(request, new Guid("5a7939fd-59de-44bd-a092-f5d8434584de"));
+
+            // Assert
+            var okObjectResult = Assert.IsType<OkObjectResult>(response);
+            Assert.IsType<Resume>(okObjectResult.Value);
+        }
+
+        [Fact]
+        public void GetResume_Returns_NotFoundResult()
+        {
+            // Arrange
+            var request = this.mockHttpContext.Request;
+            this.mockService.Setup(service => service.GetResume(Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"))).Equals(null);
+
+            // Act
+            var response = this.getResume.Run(request, new Guid("5a7939fd-59de-44bd-a092-f5d8434584de"));
+
+            // Assert
+            var notfountObjectResult = Assert.IsType<NotFoundObjectResult>(response);
```

Comment:
```suggestion
            var notFoundObjectResult = Assert.IsType<NotFoundObjectResult>(response);
```

Reply:
Solved

Reply:
Cris, please can you review my pull request? Please